spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/db/AbstractDataflowTests.java

```diff
@@ -169,7 +169,7 @@ protected List<ClusterContainer> getSkipperContainers() {
 	protected List<ClusterContainer> getDataflowContainers() {
 		ArrayList<ClusterContainer> containers = new ArrayList<>(DATAFLOW_CONTAINERS);
 		containers.add(ClusterContainer.from(TagNames.DATAFLOW_main, DATAFLOW_IMAGE_PREFIX + getDataflowLatestVersion()));
-		List<Integer> jdkTags = Arrays.asList(8, 11, 17);
+		List<Integer> jdkTags = Arrays.asList(17, 21);
```

Comment:
Kicked the tires and noticed that a couple tests fail.
We do need to add Java 21 to the container list here: https://github.com/spring-cloud/spring-cloud-dataflow/blob/main-3/spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/db/AbstractDataflowTests.java#L159-L166. Else we have tests that will fail.

Reply:
Fixed